src/styles/components/fields/_field.scss

```diff
@@ -70,7 +70,7 @@
       opacity: 1;
       svg {
         path {
-          fill: var(--color-sienna);
+          //fill: var(--color-sienna);
         }
```

Comment:
erase it instead of commenting out?

Reply:
oh whoops! missed this -- was just testing when I wasn't sure where that red was coming from. Will fix.